File: web3rpc/rpc-specs/code-samples/python/debug/blockchainInspection/setHead.py
from web3 import Web3
from web3py_ext import extend

host = "https://public-en-kairos.node.kaia.io"

number = "0x100"

w3 = Web3(Web3.HTTPProvider(host))
debug_response = w3.debug.set_head(number)

print(debug_response)

File: web3rpc/rpc-specs/code-samples/javascript/debug/blockchainInspection/setHead.js
const { JsonRpcProvider } = require("@kaiachain/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-kairos.node.kaia.io");
  const number = "0x100";

  provider.debug
    .setHead(number, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();
